API/controllers/comment-controller.js

```diff
@@ -33,3 +37,15 @@ exports.postComment = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.deleteComment = (req, res, next) => {
+  const { comment_id } = req.params;
+  return deleteComment(comment_id)
+    .then(({ rows }) => {
+      if (rows.length === 0)
```

Comment:
This check for errors and `Promise.reject` should be in the model rather than the controller